pom.xml

```diff
@@ -60,38 +60,54 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-		<alfresco.version>5.2.g</alfresco.version>
```

Comment:
why removing the alfresco.version ? it can't work without an alfresco.version used below

Reply:
Do you think it is possible to update alfresco.version e.g. to 23.2?

Reply:
This alfresco.version is used for the version of alfresco-platform-distribution in the pom.xml below. There is no recent version of this artifact. Alfresco sdk may be updated first to 4.x (in-process sdk) and perhaps alfresco-platform-distribution should be replaced in the pom.xml by alfresco-content-services-community-distribution (ACS 6 and then ACS 23).
See https://docs.alfresco.com/content-services/latest/develop/sdk/
I see that there are some changes with sdk 4.x and that https://github.com/Alfresco/alfresco-sdk-samples/ has not changed in 9 years unfortunately.

Anyway, all these changes seem not necessary at the moment to build the plugin and it works already in Alfresco 23 after the commit https://github.com/javamelody/alfresco-javamelody/commit/0305b9f2b1ab84d97615ffea67f5b2a5e052f010

If you can suggest a new pull request for this migration and if you can explain why you are doing the changes in the new PR, we may upgrade that version.

Reply:
@evernat I have created a new pull request that uses SDK 4.9. In pom.xml now you can use 23.x in <alfresco.version>. 

Although - at the moment - I do not see any additional value for this update, your previous commit works well with Alfresco 23. SoI'll leave it up to you to merge my pull request into your code, or drop it.

Thanks and best regards.